applications/photo-asset-manager/README.md

```diff
@@ -43,6 +43,7 @@ End user authentication is handled using Amazon Cognito. A user pool is provisio
 
 - [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/)
 - [AWS CloudFormation (CFN)](https://aws.amazon.com/cloudformation/)
+- [Amazon Rekognition](https://aws.amazon.com/rekognition/)
```

Comment:
Same here--I think we have a standard of linking to docs pages instead of marketing pages. Or was there a decision about this that I missed?

Reply:
Nope, just copy/pasted the wrong source.